cmd/query/app/grpc_handler.go

```diff
@@ -109,7 +114,16 @@ func (g *GRPCHandler) ArchiveTrace(ctx context.Context, r *api_v2.ArchiveTraceRe
 	if r.TraceID == (model.TraceID{}) {
 		return nil, errUninitializedTraceID
 	}
-	err := g.queryService.ArchiveTrace(ctx, r.TraceID)
+	query := spanstore.GetTraceParameters{
+		TraceID: r.TraceID,
+	}
+	if r.StartTime != nil {
```

Comment:
inconsistent with other requests - do we need to regenerate ArchiveTraceRequest with nullable=false?